### PR TITLE
Update the matrix to build the plugin with master and jruby 9k

### PR DESCRIPTION
**Pull request by ph**
This will be done with a mass update scripts.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,19 +2,15 @@
 sudo: false
 language: ruby
 cache: bundler
-env:
+env: 
 rvm:
-  - jruby-1.7.25
+- jruby-1.7.25
```
Comment (jakelandis) [inline .travis.yml]: should this be jruby 9k ?

Also if make this change, do we need 3 builds ?  It seems that the default (env blank) would end up being the same as LOGSTASH_BRANCH=master.

Comment (ph) [inline .travis.yml]: We actually still need 3 build, if you look at the job description:

screenshot 2017-06-13 08 36 23

The first job is;

- Test with JRuby 1.7.25 + released gems on rubyges, this build translate into the Logstash 5.4.1 release (current released stable).
- Test with JRuby 1.7.25 with the 5.x branch, which is the next 5.4.x release.
- The last test with master with JRuby 9k.

Comment (jakelandis) [inline .travis.yml]: Thanks for explaining the difference.

**Comment by jakelandis** [comment]
LGTM

**Comment by ph** [comment]
This will be done in a mass-update.

**Pull request closed without merging**